### PR TITLE
Polish and optimize desktop background

### PR DESCRIPTION
**Pull request by alanorozco**
1. Ignores images hidden by media query when finding background source. In some cases, hidden images would be loaded by the background, causing a delay between correct image load and background image load.

2. Fixes blur bleed.

4. Sets background color to be the same as the `amp-story-page` element in case no image is present or when it fails to load.

3. Waits for image to load before swap. On timeout, it will show the background color.

4. Previous changes allow for crossfade transition to be re-enabled, with some changes in behavior.

### CODE REVIEW

File: extensions/amp-story/0.1/amp-story-desktop.css [hunk 1]
```diff
@@ -62,20 +63,22 @@ amp-story[standalone][desktop] {
...
+  right: -150px !important;
+  bottom: -150px !important;
+  left: -150px !important;
   opacity: 0 !important;
   filter: blur(50px) !important;
-  margin: -60px !important;
   background-size: cover !important;
   background-color: transparent !important;
   background-position: center center !important;
+  will-change: opacity, z-index !important;
```
Comment (prateekbh) [inline extensions/amp-story/0.1/amp-story-desktop.css]: are we trying to animate opacity?
last time we did this, opacity animation + page movement tend to get a little janky. please check

File: extensions/amp-story/0.1/amp-story-desktop.css [hunk 2]
```diff
@@ -62,20 +63,22 @@ amp-story[standalone][desktop] {
 
 .i-amphtml-story-background {
   position: absolute !important;
-  top: 0 !important;
-  right: 0 !important;
-  bottom: 0 !important;
-  left: 0 !important;
+  top: -150px !important;
```
Comment (prateekbh) [inline extensions/amp-story/0.1/amp-story-desktop.css]: because?

Comment (alanorozco) [reply extensions/amp-story/0.1/amp-story-desktop.css]: Blur bleed :)

File: extensions/amp-story/0.1/amp-story-desktop.css (hunk 1 above)
Comment (alanorozco) [reply extensions/amp-story/0.1/amp-story-desktop.css]: Confirmed as working, issues were caused by lack of transition sync/two transitions at the same time.

**Pull request merged**